Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.h

```diff
@@ -112,6 +112,7 @@ class UnifiedPDFPlugin final : public PDFPluginBase, public WebCore::GraphicsLay
 #endif
     enum class IsAnnotationCommit : bool { No, Yes };
     static OptionSet<RepaintRequirement> repaintRequirementsForAnnotation(PDFAnnotation *, IsAnnotationCommit = IsAnnotationCommit::No);
+    void repaintAnnotationsForFormField(RetainPtr<NSString> fieldName);
```

Comment:
I don't think we need to pass in a retainptr, it's on the caller to keep it alive